pom.xml

```diff
@@ -135,10 +145,39 @@
                     <username>git</username> <!-- Make sure git-push is not tried with local user -->
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>com.github.wajda</groupId>
+                <artifactId>scala-cross-build-maven-plugin</artifactId>
```

Comment:
If you are cross-building and want it on maven you should change the artifactId to `hyperdrive_2.11` to have that `_2.12` or `_2.11` changed in name. And the same in modules. Not sure if I am remembering this right.

Reply:
That's absolutely correct. Totally forgot it. Thanks for pointing out